heroku.yml

```diff
@@ -0,0 +1,14 @@
+setup:
+ addons:
+   - plan: heroku-postgresql
+build:
+ docker:
+   web: docker pull openmrs/openmrs-reference-application-distro:latest
```

Comment:
what happens when you docker compose up .in this directory

Reply:
do you mean to just have the docker file within the repo  and point to its location instead?